examples/carplay-web-app/src/App.tsx

```diff
@@ -26,10 +26,12 @@ const config: Partial<DongleConfig> = {
   width,
   height,
   fps: 60,
-  mediaDelay: 0,
+  mediaDelay: 300,
```

Comment:
Was configuring the dongle in its own web interface, and it actually errors out when sending anything below 300 - aligning with the minimum there.